darwin/Platform.c

```diff
@@ -292,9 +292,9 @@ void Platform_setMemoryValues(Meter* mtr) {
 
    mtr->total = dhost->host_info.max_mem / 1024;
    mtr->values[MEMORY_METER_USED] = (double)(vm->active_count + vm->wire_count) * page_K;
-   mtr->values[MEMORY_METER_BUFFERS] = (double)vm->purgeable_count * page_K;
    // mtr->values[MEMORY_METER_SHARED] = "shared memory, like tmpfs and shm"
    // mtr->values[MEMORY_METER_COMPRESSED] = "compressed memory, like zswap on linux"
+   mtr->values[MEMORY_METER_BUFFERS] = (double)vm->purgeable_count * page_K;
```

Comment:
Note that none of the Platform changes are any longer functionally required, unlike the original version of the patch, thanks to the `MEMORY_METER_` defines.

But they're still here as it seems worth keeping the order consistent everywhere.

Reply:
Those order changes seem good. Now with the constants for the various fields you at least have no problem with consistency even if changing around the order.